tyr-runner/src/main/java/org/jboss/tyr/check/DescriptionCheck.java

```diff
@@ -48,6 +52,19 @@ public String check(JsonObject payload) {
                         break;
                     }
                 }
+                for (OptionalPatterns optionalRow : optionalRows){
+                    Matcher preconditionMatcher = optionalRow.getPrecondition().matcher(line);
+                    if (preconditionMatcher.matches()){
+                        Matcher patternMatcher = optionalRow.getPattern().matcher(line);
+                        if (!patternMatcher.matches()){
+                            return optionalRow.getMessage();
+                        }
+                        else {
+                            optionalRows.remove(optionalRow);
+                            break;
```

Comment:
this break is redundant as it's the last statement in the loop.

Reply:
Isn't the same case on line 52 too?

Reply:
my bad, the intent is to finish when you find a matching row. Resolved.